fern/ir-types-v30/definition/api.yml

```diff
@@ -0,0 +1,4 @@
+name: ir
+docs: |
+  Changes in this version of IR include:
+  - Several improvements to Example types, including the new ExampleUnionType.
```

Comment:
would you be up to provide a little more detail here? the ts generator will need to be upgraded so would be nice to have some more idea